### ISSUE TITLE
Auto Layout Error

### ISSUE THREAD

**Issue by resultsux** [BuilderIO/figma-html#108]
**Describe the bug**
I have autolayout on the component, but the plugin keeps throwing errors saying I don't.

**To Reproduce**
1.Select a component
2.Run plugin
3.Click "Generate Code"

**Expected behavior**
I'd expect it to generate the react code/styling for the component
![Screen Shot 2022-06-07 at 11 17 59 AM](https://user-images.githubusercontent.com/65322208/172431313-53a4c331-fa9d-4349-b21b-4d0a8d82b45f.png)

**Additional context**

**Comment by samijaber** [comment]
@resultsux Can you share a file containing the Figma design? Or a link to the Figma design?

**Comment by resultsux** [comment]
Yes. You need to turn everything into components. I believe turned on auto
layout. But that wasn’t enough. You must turn it into a component by right
clicking on it and selecting create component. Try that and let me know
what you happens.

**Comment by kkharji** [comment]
https://www.figma.com/file/AULqU413s2kLXGzG4yjE4p/PavBrandWebsite?node-id=28%3A62

Right now it marks everything with the layout errors
